app/javascript/components/admin/manage_users/InvitedUsers.jsx

```diff
@@ -6,53 +6,66 @@ import { CheckIcon, XMarkIcon } from '@heroicons/react/24/solid';
-          </tr>
-        </thead>
-        <tbody className="border-top-0">
-          {invitations?.data?.length
-            ? (
-              invitations?.data?.map((invitation) => (
-                <tr key={invitation.email} className="align-middle text-muted">
-                  <td className="text-dark border-0">{invitation.email}</td>
-                  <td className="text-dark border-0">{invitation.updated_at}</td>
-                  <td className="text-dark border-0">
-                    { invitation.valid ? <CheckIcon className="text-success hi-s" /> : <XMarkIcon className="text-danger hi-s" />}
-                  </td>
+    <div>
+      {
+      (searchInput && invitations?.data.length === 0)
+        ? (
+          <div className="mt-5">
+            <NoSearchResults name={t('user.users')} searchInput={searchInput} />
+          </div>
+        ) : (
+          <div id="admin-table">
+            <Table className="table-bordered border border-2 mb-0" hover responsive>
+              <thead>
+                <tr className="text-muted small">
+                  <th className="fw-normal border-end-0">{ t('user.email_address') }<SortBy fieldName="email" /></th>
+                  <th className="fw-normal border-0">{ t('admin.manage_users.invited.time_sent') }</th>
+                  <th className="fw-normal border-0">{ t('admin.manage_users.invited.valid') }</th>
                 </tr>
-              ))
-            )
-            : (
-              <tr>
-                <td className="fw-bold">
-                  { t('user.no_user_found') }
-                </td>
-              </tr>
-            )}
-        </tbody>
-      </Table>
-      <div className="pagination-wrapper">
-        { invitations?.meta && (
-        <Pagination
-          page={invitations?.meta?.page}
-          totalPages={invitations?.meta?.pages}
-          setPage={setPage}
-        />
-        ) }
-      </div>
+              </thead>
+              <tbody className="border-top-0">
+                {invitations?.data?.length
+                  ? (
+                    invitations?.data?.map((invitation) => (
+                      <tr key={invitation.email} className="align-middle text-muted">
+                        <td className="text-dark border-0">{invitation.email}</td>
+                        <td className="text-dark border-0">{invitation.updated_at}</td>
+                        <td className="text-dark border-0">
+                          { invitation.valid ? <CheckIcon className="text-success hi-s" /> : <XMarkIcon className="text-danger hi-s" />}
+                        </td>
+                      </tr>
+                    ))
+                  )
+                  : (
+                    <tr>
+                      <td className="fw-bold">
+                        { t('user.no_user_found') }
+                      </td>
+                    </tr>
+                  )}
```

Comment:
We should replace this for a "EmptyResource" card eventually. Like we have in Rooms, Recordings, SharedAccess, Presentation, when there is no resource yet.

Reply:
Yeah i'll do that in another upcoming PR